josemak25/hairx-client
# Fix get started

#### What does this PR do?

Have the logic of the get started screen slider functioning properly.

#### Description of Task to be completed?

Have the get start screen show the following:

1. A title of the current slide.
2. And skip and next clickable button with action attached to it.
3. And lastly login clickable button with action attached to it that routes you to login page.


#### How should this be manually tested?

After cloning the repo,
`CD` into it and `RUN`

javascript
git checkout fix-get-started

git pull origin fix-get-started

yarn install

yarn test

Using an emulator: Launch the app, and you should see the home screen after loading.

#### Screenshots (if appropriate)

![Simulator Screen Shot - iPhone 11 Pro Max - 2020-04-21 at 15 05 32](https://user-images.githubusercontent.com/33870502/79875506-a9ed4180-83e1-11ea-891c-4845754f4bd0.png)
